src/Spice86.Core/CLI/Configuration.cs

```diff
@@ -163,4 +163,7 @@ public sealed class Configuration {
     /// </summary>
     [Option(nameof(AudioEngine), Default = AudioEngine.PortAudio, Required = false, HelpText = "Audio engine to use. Values are PortAudio or Dummy")]
     public AudioEngine AudioEngine { get; init; }
+
+    [Option(nameof(Xms), Default = false, Required = false, HelpText = "Enable XMS")]
```

Comment:
Would it harm to enable XMS by default, with the same reason as to why you enabled EMS by default?

Reply:
Not much harm once it works. :)